example/.yalc/react-native-feed-media-audio-player/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### 4.0.3 (2023-02-06)
```

Comment:
Ah good catch, I was only updating the changelog for the main React SDK package/changelog